lib/drivers/animasDriver.js

```diff
@@ -1775,6 +1776,19 @@ module.exports = function (config) {
       cfg.tzoUtil.fillInUTCInfo(record, event.jsDate);
       record = record.done();
       postrecords.push(record);
+
+      // Since Animas does not generate a separate SMBG event for Vibe, we use the
+      // calibration event from the pump to also generate a manual SMBG record
+      // in order to display it in Blip
+      var bgRecord = cfg.builder.makeSMBG()
+        .with_deviceTime(event.deviceTime)
+        .with_subType('manual')
+        .with_value(event.value)
+        .with_units('mg/dL')
```

Comment:
According to the spec, yes. Would love to see if that is indeed the case on a `mmol/L` pump too, but we don't have any to verify.